### ISSUE TITLE
Missing fonts in wavemon?

### ISSUE THREAD

**Issue by sliddjur** [uoaerg/wavemon#50]
Hello I am missing some fonts when I start wavemon. I am using fresh install of Ubuntu 18.04 and both gnome terminal and tilix terminal shows the same font error (I suppose).

![image](https://user-images.githubusercontent.com/10002057/40354121-74fad7c2-5db3-11e8-9426-338264b70d4c.png)

**Comment by grrtrr** [comment]
Troubleshooting steps:
- check locale settings: https://help.ubuntu.com/community/Locale
- ncurses requires ANSI escape sequences, your $TERM should support this (known working examples are xterm and vt100)
- try running this from xterm, after checking locale.

**Comment by grrtrr** [comment]
@sliddjur - has this been resolved, did you find a fix?

**Comment by sliddjur** [comment]
@grrtrr I installed xterm from apt. It still looks the same. 
```olof@olof-ubuntu ~> echo $TERM
xterm-256color```

In xterm app, $TERM is xterm.

My locale
```olof@olof-ubuntu ~> locale -a
C
C.UTF-8
en_AG
en_AG.utf8
en_AU.utf8
en_BW.utf8
en_CA.utf8
en_DK.utf8
en_GB.utf8
en_HK.utf8
en_IE.utf8
en_IL
en_IL.utf8
en_IN
en_IN.utf8
en_NG
en_NG.utf8
en_NZ.utf8
en_PH.utf8
en_SG.utf8
en_US.utf8
en_ZA.utf8
en_ZM
en_ZM.utf8
en_ZW.utf8
nb_NO.utf8
POSIX
sv_FI.utf8
sv_SE.utf8
```

**Comment by grrtrr** [comment]
Thanks for getting back. I still think that this related to locale settings, of which there are quite a few.

In the above picture the 'funny' characters appear between
- thousand separator for _RX/TX_ (220,293 and 226,402 respectively);
- thousand separator for _retries_ (number seems to be 2,904 but is cut off -- seems a lot).

The _numerical_ setting is set via the `LC_LOCALE`, which is in turn affected by other settings.

In addition, instead of a dot (`"."`) the dBm setting above uses a _comma_ ("1,26nW"), which is unusual. This too would indicate an unusual locale setting.

Under the hood, this is all done using the [`setlocale()`](https://linux.die.net/man/3/setlocale) call.

All of these settings can be bypassed by setting `LC_ALL`.

_Can you please try this_: `export LC_ALL=C`, and then run `wavemon`? 

Likely you will then have no thousand-separator between numbers. If that works I can take it from there, maybe the use of `setlocale` can be improved.

**Comment by sliddjur** [comment]
yes, if I set export LC_ALL=C and run wavemon, it works.

**Comment by grrtrr** [comment]
Thank you. I will work on a fix that does not require setting an environment variable. The downside is that it will use US/GB numeric separator (a `.` between 1 and fractions of 1, and a `,` between groups of thousands). This may look strange in other countries, but is still better than the funny fonts - which might have to do with the ncurses library.

**Comment by grrtrr** [comment]
As promised - if you are able to compile from `master`, could you give the above a quick run?

It should resolve the locale-specific issues. It is now locale-independent.

**Comment by sliddjur** [comment]
![image](https://user-images.githubusercontent.com/10002057/45030298-af9ae580-b04b-11e8-8dde-8ae1abaff4ad.png)

It looks better now.

```
olof@olof-ubuntu /opt/wavemon % /usr/local/bin/wavemon -v
wavemon 0.8.2
Distributed under the terms of the GPLv3.
```

**Comment by grrtrr** [comment]
Resolved in 324b1516545b569cdbd43b25d3745ffb0095c81a
